pkg/controller/migmigration/task.go

```diff
@@ -41,6 +41,7 @@ const (
 	EnsureFinalRestore            = "EnsureFinalRestore"
 	FinalRestoreCreated           = "FinalRestoreCreated"
 	FinalRestoreFailed            = "FinalRestoreFailed"
+	Verify                        = "Verify"
```

Comment:
The name `Verification` seems more like phase name.  Eg.  _"The migration is in the verification phase."_

Reply:
Updated as well.